### ISSUE TITLE
Cannot reproduce Imagenet Eval with provided checkpoints EfficientNetV2 BX variants.

### ISSUE THREAD

**Issue by sebastian-sz** [google/automl#1101]
Hello, I'm trying to reproduce the Imagenet Validation Accuracy with the checkpoints in this repository.

I'm noticing slight differences in reported Top1 accuracy:
| Model variant | Reported in this repo| My code |
| ------- | ----- | ----     |
|  B0 | 78.7 | 78.5 |
|  B1 | 79.8 | 79.6 |

I'm running eval with the following snippet:

```python
# !pip install tensorflow_addons tensorflow_datasets pyyaml

import tensorflow as tf
import tensorflow_datasets as tfds

import effnetv2_model
import preprocessing

MODEL = 'efficientnetv2-b0'
INPUT_SHAPE = 224


def preprocess_fn(image, label):
    augname = 'effnetv1_autoaug'  # V1 preprocessing for Bx variants
    bytes = tf.io.encode_jpeg(image)  # V1 preprocessing expects bytes
    image = preprocessing.preprocess_image(
        bytes, INPUT_SHAPE, is_training=False, augname=augname
    )
    return image, tf.one_hot(label, 1000)

def main():
    # Load model
    model = effnetv2_model.get_model(MODEL, training=False)
    model.compile(metrics=[
        tf.keras.metrics.TopKCategoricalAccuracy(k=1, name="top1"),
        tf.keras.metrics.TopKCategoricalAccuracy(k=5, name="top5")
    ])

    # Load data
    ds = tfds.load('imagenet2012', split='validation', as_supervised=True)
    ds = ds.map(preprocess_fn, num_parallel_calls=tf.data.AUTOTUNE)
    ds = ds.batch(8).prefetch(tf.data.AUTOTUNE)

    # Run eval:
    summary = model.evaluate(ds, return_dict=True)
    print(summary)  # 'top1': 0.7850, 'top5': 0.9427



if __name__ == "__main__":
    main()
```
Am I doing something wrong during the evaluation?
